lockc/src/runc.rs

```diff
@@ -380,7 +377,100 @@ impl RuncWatcher {
             }
         }
 
-        Ok(RuncWatcher { fd })
+        Ok(RuncWatcher {
+            bootstrap_rx,
+            ebpf_tx,
+            fd,
+        })
+    }
+
+    async fn add_container(
+        &self,
+        container_id: String,
+        pid: i32,
+        policy_level: container_policy_level,
+    ) -> Result<(), HandleRuncEventError> {
+        let (responder_tx, responder_rx) = oneshot::channel();
+
+        self.ebpf_tx
+            .send(EbpfCommand::AddContainer {
+                container_id,
+                pid,
+                policy_level,
+                responder_tx,
+            })
+            .await?;
+        responder_rx.await??;
+
+        Ok(())
```

Comment:
It feels link we don't need this line, the `responder_rx.await` is already returning a `Result`